test/ring/middleware/oauth2_test.clj

```diff
@@ -2,10 +2,10 @@
   (:require [clj-http.fake :as fake]
             [clj-time.core :as time]
             [clojure.string :as str]
-            [clojure.test :refer :all]
+            [clojure.test :refer [deftest is testing]]
```

Comment:
this is just my personal taste to make the linter happy and not have magic symbols, but I can revert if you don't like it

Reply:
Please revert the change. I don't mind commits that fix formatting, but they should be reserved for their own PRs. By adding formatting changes to other commits we just pollute the diff and make it hard to understand what's changed.

However, feel free to send me another PR with *just* formatting changes, and I'll merge it in.